src/main/resources/avro/metadata.avdl

```diff
@@ -122,10 +122,25 @@ For data accessors, they're a simple way to scope exploration and analysis
 what's the distribution of that result in the data from our project?)   
 */
 record Dataset {
-  /** The dataset UUID. This is globally unique. */
+  /**
+  The dataset's id. This is the local/legacy/generic id.
+  */
   string id;
 
-  /** A description of the dataset. */
+  /**
+  The dataset's guid. This is globally unique id (UUIDv4).
+  TODO: Required?!
+  */
```

Comment:
agreed @dglazer, a GUID system needs to be consistent across all
GA4GH object in order to implement a provenance system.


David Glazer <notifications@github.com> writes:
> In src/main/resources/avro/metadata.avdl:
> 
> >    string id;
> >
> > -  /** A description of the dataset. */
> > +  /**
> > +  The dataset's guid. This is globally unique id (UUIDv4).
> > +  TODO: Required?!
> > +  */
> 
> I agree -- I like the idea of an API-wide GUID strategy; let's not go piecemeal
> until we have one.
> 
> —
> Reply to this email directly or view it on GitHub.*
>